Cargo.lock

```diff
@@ -2605,9 +2629,9 @@ dependencies = [
 
 [[package]]
 name = "jwt-simple"
-version = "0.11.2"
+version = "0.11.1"
```

Comment:
I think you rolled back a few too many dependency updates, could you reapply a `cargo update` and see what sticks?

Reply:
sure!

Reply:
Fixed in ad17de71

Reply:
I think this stems from deno pinning their dependency on serde